app/root/root.css

```diff
@@ -1121,11 +1121,27 @@ code .comment {
   cursor: pointer;
 }
 
+.artifact-line {
+  display: flex;
+}
+
 .artifact-name {
   font-weight: 600;
   text-decoration: underline;
   cursor: pointer;
-  display: block;
+}
+
+.artifact-view {
+  font-weight: 600;
+  display: flex;
+  align-items: center;
```

Comment:
(woops, my comment didn't make it through)

nit: maybe align to `baseline` so that the "View" text lines up vertically with the artifact link? (Might need to tweak the code icon manually to get it to look nice)

Reply:
Looks like the issue was the lack of a height constraint on the icon and some margins - fixed.